### PR TITLE
feat: fixtures added

### PR DESCRIPTION
**Pull request by gluhovSemen**

### CODE REVIEW

File: fixtures/user_profile_fixture.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
```
Comment (egorvavilov) [inline fixtures/user_profile_fixture.json:1]: use copypaste in file to create user profiles for each user

File: fixtures/category_fixture.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+[
```
Comment (egorvavilov) [inline fixtures/category_fixture.json:1]: Rename Categories too, make them real, together with descriptions. Use Chat GPT if you are too lazy to write text descriptions
;)

File: docker-compose.yml [hunk 3]
```diff
@@ -27,5 +28,6 @@ services:
       - POSTGRES_PASSWORD
       - POSTGRES_DB
       - POSTGRES_USER
+      - DB_HOST
     ports:
-      - '5432:5432'
\ No newline at end of file
+      - '5433:5432'
```
Comment (egorvavilov) [inline docker-compose.yml]: why? how it helps current ticket?

**Review by egorvavilov — CHANGES_REQUESTED** [review]

File: fixtures/user_profile_fixture.json (hunk 1 above)
Comment (gluhovSemen) [reply fixtures/user_profile_fixture.json:1]: done

File: fixtures/category_fixture.json (hunk 2 above)
Comment (gluhovSemen) [reply fixtures/category_fixture.json:1]: I hope now its enough

File: docker-compose.yml (hunk 3 above)
Comment (gluhovSemen) [reply docker-compose.yml]: My bad was not working local cos i had local DB running on this port, but i think it wont work for everybody who is using both Docker and local

**Review by egorvavilov — APPROVED** [review]

**Pull request merged**